contracts/contracts/Poll.sol

```diff
@@ -144,7 +144,9 @@ contract Poll is Params, Utilities, SnarkCommon, Ownable, EmptyBallotRoots, IPol
     }
 
     // init messageAq here by inserting placeholderLeaf
-    uint256[2] memory dat = [NOTHING_UP_MY_SLEEVE, 0];
+    uint256[2] memory dat;
+    dat[0] = NOTHING_UP_MY_SLEEVE;
+    dat[1] = 0;
```

Comment:
Not sure what's the difference?

Reply:
So, inline assignment requires more gas than separate one. This is also true for struct assignment.

Reply:
Actually, I found it here https://github.com/0xisk/awesome-solidity-gas-optimization in the articles section